src/IntegrationTests/IncludeMembers.cs

```diff
@@ -920,22 +1536,45 @@ public class Destination
         public int Code { get; set; }
         public int OtherCode { get; set; }
     }
+    public class DestinationA : Destination
+    {
+        public string A { get; set; }
+    }
+    public class DestinationB : Destination
+    {
+        public string B { get; set; }
+    }
     public class Context : LocalDbContext
     {
         public DbSet<Source> Sources { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+            modelBuilder.Entity<SourceA>();
+            modelBuilder.Entity<SourceB>();
+        }
     }
     public class DatabaseInitializer : DropCreateDatabaseAlways<Context>
     {
         protected override void Seed(Context context)
         {
-            var source = new Source { Name = "name" };
+            var sourceA = new SourceA { Name = "name1", A = "a", };
+            context.Sources.Add(sourceA);
+            var sourceB = new SourceB { Name = "name2", B = "b", };
+            context.Sources.Add(sourceB);
+            var source = new Source { Name = "name3" };
             context.Sources.Add(source);
             base.Seed(context);
         }
     }
     protected override MapperConfiguration CreateConfiguration() => new(cfg =>
     {
-        cfg.CreateProjection<Source, Destination>().IncludeMembers(s => s.InnerSources.FirstOrDefault(), s => s.OtherInnerSources.FirstOrDefault());
+        cfg.CreateMap<Source, Destination>().IncludeMembers(s => s.InnerSources.FirstOrDefault(), s => s.OtherInnerSources.FirstOrDefault())
+            .Include<SourceA, DestinationA>()
+            .Include<SourceB, DestinationB>();
+        cfg.CreateMap<SourceA, DestinationA>();
+        cfg.CreateMap<SourceB, DestinationB>();
```

Comment:
None of these derived maps actually have included members. I want to see at least two such cases, one for a simple included member, chained properties, and one that uses the let clause, such as this one, with a method. 
Check [the execution plan](https://docs.automapper.org/en/latest/Understanding-your-mapping.html). That extension is very useful :)

Reply:
Unless I've missed smth, you didn't address this and a similar comment for the subquery test.

Reply:
yes will do today. was busy with the family over the weekend :)

Reply:
I extended the tests. basicaly doing the same thing as for the base class in the derived class. that uncovered some problems with the current approach. I was able to fix some of the issues but there is at least one more issues remaining.  im not quite certain whats the issue as the expression it self looks ok but the entity framework transformation of the query cannot be translated to sql because it contains a a lamda that should probably be a property expression.

Reply:
my current thinking is that it would be better to have multiple intermediate let statements that would also improve expression caching.

e.g:

instead of:
```c#
.Select(s => new {
    Source = s,
    Inner = s.InnerSources.FirstOrDefault(),
    Inner#Details = s.InnerSources.FirstOrDefault().InnerSourceDetails.FirstOrDefault().
})
```
generate:
```c#
.Select(s => new {
    Source = s,
    Inner = s.InnerSources.FirstOrDefault()
}).Select(s => new {
    s.Source,
    s.Inner,
    Details = s.Inner.InnerSourceDetails.FirstOrDefault()
})
```

Reply:
One let clause should be enough for everyone :) It's not clear to me why EF would not handle the first expression. Except that it's too complicated. And if that's the case, just simplify your test so things don't compose. But of course I still want to see a subquery with a method in a derived class.

Reply:
im investigating